target/linux/ramips/dts/mt7621_beeline_smartbox-pro.dts

```diff
@@ -0,0 +1,332 @@
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset Button";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "WPS Button";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		switch_bt {
+			label = "ROUT<->REP Button-switch";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			linux,input-type = <EV_SW>;
+			linux,code = <BTN_0>;
+			debounce-interval = <60>;
+		};
+	};
+};
+
+/*&keys { // Example
+	wifi { 
+		label = "Wi-Fi button";
+		linux,code = <KEY_RFKILL>;
+		gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+	};
+	switch_b0 { 
+		label = "switch_b0";
+		linux,input-type = <EV_SW>;
+		linux,code = <BTN_0>;
+		gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+		debounce-interval = <60>;
+	}; 
+		led_switch {
+			label = "led_switch";
+			gpios = <&gpio 13 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_LIGHTS_TOGGLE>;
+			linux,input-type = <EV_SW>;
+		};
+};*/
+
+&nand { // 256 MiB
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "BootLoader";
+			reg = <0x0 0x100000>;
+			read-only;
+
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x80000>;
+				read-only;
+			};
+
+			partition@80000 {
+				label = "BootLoad-Env";
+				reg = <0x80000 0x20000>;
+			};
+		};
+
+		factory: partition@100000 {
+			label = "Factory"; // RF-EEPROM
+			reg = <0x100000 0x100000>;
+			read-only;
+		};
+
+		partition@200000 {
+			label = "sys_data";
+			reg = <0x200000 0x1400000>;
+			read-only;
+		};
+
+		partition@1600000 {
+			label = "boot_flag";
+			reg = <0x1600000 0x100000>;
+			read-only;
+		};
+
+		partition@1700000 {
+			label = "kernel"; // Kernel_1
+			reg = <0x1700000 0x400000>;
+		};
+
+		partition@1b00000 {
+			label = "Kernel_2";
+			reg = <0x1b00000 0x400000>;
+			read-only;
+		};
+
+		partition@1f00000 {
+			label = "ubi"; // RootFS_1
+			reg = <0x1f00000 0x1e00000>;
+		};
+
+		partition@3d00000 {
+			label = "RootFS_2";
+			reg = <0x3d00000 0x1e00000>;
+			read-only;
+		};
+
+		partition@5b00000 {
+			label = "JVM/OSGI1";
+			reg = <0x5b00000 0x3200000>;
+			read-only;
+		};
+
+		partition@8d00000 {
+			label = "JVM/OSGI2";
+			reg = <0x8d00000 0x3200000>;
+			read-only;
+		};
+
+		partition@bf00000 {
+			label = "OSGI data";
+			reg = <0xbf00000 0x3c00000>;
+			read-only;
+		};
+
+		partition@fb00000 {
+			label = "Ftool";
+			reg = <0xfb00000 0x100000>;
+			read-only;
+		}; // fc00000 or ff80000 bbt 0x80000?
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi@0,0 { // MT7612EN
+		compatible = "mediatek,mt76";
+		reg = <0x0 0 0 0 0>;
+		ieee80211-freq-limit = <5000000 6000000>;
+		mediatek,mtd-eeprom = <&factory 0x8000>;
+
+		nvmem-cells = <&macaddr_factory_21000>;
+		nvmem-cell-names = "mac-address";
+		mac-address-increment = <3>;
```

Comment:
Please add a MAC address overview to the commit message.
https://openwrt.org/docs/guide-developer/device-support-policies#mac_addresses

Reply:
Added to commit. Also described here.
https://forum.openwrt.org/t/adding-support-for-sercomm-s1500-clones-beeline-smartbox-pro-wifire-s1500-nbn/110065/15?u=maxs0nix